ArmRobot/Assets/Scripts/RandomizerPositionObject.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizerPositionObject : MonoBehaviour
```

Comment:
What's the difference between this class and `TablePositionRandomizer`? Can they be combined? Seems like they are both for randomly positioning objects on the table. There should be just one class for this, which knows about the table and the robot but is agnostic to the types and number of objects being placed.

Reply:
So, I keep the tablePositionRandomizerCylinder file but I don't use it anymore. How it works is I have a TablePositionRandomizer to randomize only the position of the cube and then I wrote a RandomizerPositionObjects which randomize all the other objects. I did that so that we can easily go back to a situation where there is only the cube.

Reply:
A class designed to randomly position an arbitrary list of supported objects on the table ought to be able to move just one object. There is still a lot of repeated code between `TablePositionRandomizer` and `RandomizerPositionObject`. Try combining them into one class that can do both jobs.